deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/migrationConsole.yaml

```diff
@@ -86,11 +86,11 @@ spec:
     {{- if $developerModeEnabled  }}
         - name: local-console-libs
           hostPath:
-            path: /opensearch-migrations/TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib
+            path: /opensearch-migrations/migrationConsole/lib
             type: Directory
         - name: local-test-config-processor
           hostPath:
-            path: /opensearch-migrations/TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/workflow-tests/resources/scripts
+            path: /opensearch-migrations/migrationConsole/lib/console_link/tests/workflow-tests/resources/scripts
```

Comment:
This should be here for this PR, but pulling this scripts directory any futher at all is an anti-pattern - it holds some fake mock scripts that look pretty similar to the real scripts.